keps/sig-api-machinery/3157-watch-list/kep.yaml

```diff
@@ -20,16 +20,16 @@ stage: alpha
 # The most recent milestone for which work toward delivery of this KEP has been
 # done. This can be the current (upcoming) milestone, if it is being actively
 # worked on.
-latest-milestone: "v1.27"
+latest-milestone: "v1.28"
 
 # The milestone at which this feature was, or is targeted to be, at each stage.
 milestone:
-  alpha: "v1.27"
+  alpha: "v1.28"
```

Comment:
Let's leave this one as it was - we had alpha just we're extending its scope now.